packages/forklift-console-plugin/src/modules/Plans/modals/PlanCutoverMigrationModal.tsx

```diff
@@ -77,12 +82,16 @@ export const PlanCutoverMigrationModal: React.FC<PlanCutoverMigrationModalProps>
 
   const onTimeChange: (
     event: React.FormEvent<HTMLInputElement>,
-    time: string,
+    timeInput: string,
     hour?: number,
     minute?: number,
     seconds?: number,
     isValid?: boolean,
-  ) => void = (_event, _time, hour, minute) => {
+  ) => void = (_event, timeInput, hour, minute, _seconds, isValid) => {
+    setTime(timeInput);
```

Comment:
For some reason, empty time is considered valid, so you need to check if `timeInput != empty string` to avoid this:
![Screenshot from 2025-03-12 00-02-11](https://github.com/user-attachments/assets/fbbc39db-ecca-42eb-9804-54cb29417487)